config/routes.rb

```diff
@@ -13,15 +13,15 @@
   root to: "test#index"
   get "about", to: "about#index"
   get "/users/account", to: "users/account#index"
-  get "/users/account/personal_details", to: "users/account#personal_details"
+  get "/users/account/personal-details", to: "users/account#personal_details"
 
   form_handler = FormHandler.instance
   form = form_handler.get_form("2021_2022")
 
-  resources :case_logs do
+  resources :case_logs, :path => '/case-logs' do
     collection do
-      post "/bulk_upload", to: "bulk_upload#bulk_upload"
-      get "/bulk_upload", to: "bulk_upload#show"
+      post "/bulk-upload", to: "bulk_upload#bulk_upload"
+      get "/bulk-upload", to: "bulk_upload#show"
     end
 
     member do
```

Comment:
You'd want to change `get page.id.to_s, to: "case_logs##{page.id}"` here to:

`get page.id.to_s.dasherize, to: "case_logs##{page.id}"` I think. Or else change the page ids in the form definition (`# config/forms/2021_2022.json`